recipes/darr/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "darr" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 183886142de227c8179409b8c69a5dfec0c74d467f9aa78be8224070961cad1b
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
```

Part of the `noarch` suggestion above

```suggestion
    - python >=3.6
```